repo_files/documentations/line_chart.md

```diff
@@ -186,6 +186,7 @@ LineChart(
 |:-------|:----------|:------------|
 |text|text string of each row in the tooltip bubble|null|
 |textStyle|[TextStyle](https://api.flutter.dev/flutter/dart-ui/TextStyle-class.html) of the showing text row|null|
+|textAlign|[TextStyle](https://api.flutter.dev/flutter/dart-ui/TextAlign-class.html) of the showing text row|null|
```

Comment:
Please put `TextAlign.center` on the default value section. it is not null.

Reply:
Done. Good catch.